lib/src/views/Calendar/Day.tsx

```diff
@@ -24,6 +24,10 @@ export const useStyles = makeStyles(
       // background required here to prevent collides with the other days when animating with transition group
       backgroundColor: theme.palette.background.paper,
       color: theme.palette.text.primary,
+      '&:disabled': {
+        pointerEvents: 'none',
+        color: theme.palette.text.hint,
```

Comment:
Deprecated, maybe you meant?
```suggestion
        color: theme.palette.text.secondary,
```